kalite/distributed/static/js/distributed/rating/models.js

```diff
@@ -0,0 +1,18 @@
+var Backbone = require("../base/backbone");
+
+var RatingModel = Backbone.Model.extend({
+    urlRoot: "/api/content_rating"
+});
+
+var ContentRatingCollection = Backbone.Collection.extend({
+    parse: function(response) {

```

Comment:
Ah, I guess not. I assumed the `meta` object returned along with objects would throw off Backbone.